guide/glossary.md

```diff
@@ -0,0 +1,317 @@
+---
+layout: guide
+title: Glossary
+nav_order: 4
+---
+
+# Glossary
+{: .no_toc }
+
+## Address
+
+An identifier of 26-35 alphanumeric characters that can be used to receive bitcoin. There are [several address formats](https://en.bitcoin.it/wiki/List_of_address_prefixes){:target="_blank"} based on different standards and use cases. Applications should make it clear to users which standards are being used. Support for formats varies, so users may need to know this information for backup and future recovery.
+
+When users enter addresses, these formats can also be used to instantly validate correctness and possibly deduct other useful information.
+
+These are the three most common ones:
+
+
+**Legacy address - [P2PKH](https://en.bitcoin.it/wiki/Transaction#Pay-to-PubkeyHash){:target="_blank"}**<br/>
+Stands for pay-to-pubkey-hash, i.e pay to a hash of the recipient’s public key. This was the original address format for bitcoin and is sometimes called *legacy address*. Legacy addresses are not segwit compatible, but you can still send BTC from a P2PKH address to a segwit address without any problems. The average fee when sending from a P2PKH address is likely to be higher than when sending from a segwit address, however, because legacy address transactions are larger in size.
+
+*Legacy addresses* begin with the number 1 and are mixed and case sensitive. <br/>
+For example: `1BvBMSEYstWetqTFn5Au4m4GFg7xJaNVN2`
+
+**Script address - [P2SH](https://en.bitcoin.it/wiki/Pay_to_script_hash){:target="_blank"}**<br/>
+P2SH, which stands for pay-to-script-hash, was defined in [BIP13](https://github.com/bitcoin/bips/blob/master/bip-0013.mediawiki) and enables more functionality than legacy addresses. The P2SH script function is most commonly used for multisig addresses which can specify, for example, that multiple digital signatures are required to authorize the transaction. They are also used to enable non-native segwit transactions using a process known as P2WPKH-in-P2SH. This address type is widely supported and can be used to send funds to both P2PKH and Bech32 addresses.
+
+*Script addresses* start with the number 3 and are mixed and case sensitive. <br/>
+For example: `3J98t1WpEZ73CNmQviecrnyiWrnqRhWNLy`
+
+**SegWit address - [Bech32](https://en.bitcoin.it/wiki/Bech32){:target="_blank"}**<br/>
+Bech32 is the *SegWit* address format (for segregated witness), and is specified in [BIP173](https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki){:target="_blank"}. They are supported by the majority of software and hardware wallets, but a minority of exchanges. While most exchanges enable sending funds to bech32 addresses, they don’t enable users to receive them with this format. Bech32 adoption by wallet-applications and exchanges is tracked on [this page](https://en.bitcoin.it/wiki/Bech32_adoption).
```

Comment:
`Bech32 is the *SegWit* address format`.  This always confuses me. Is it a Bech32 address or a SegWit address? Or both? Can we pick one?

Reply:
`they don’t enable users to receive them with this format`. Curious if there's a reason for this.

Reply:
Simplified.